instrumentation/elasticsearch/elasticsearch-rest-7.0/library/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
 dependencies {
   library("org.elasticsearch.client:elasticsearch-rest-client:7.0.0")
   implementation("net.bytebuddy:byte-buddy")
-  implementation(project(":instrumentation:elasticsearch:elasticsearch-rest-common:library"))
+  implementation(project(":instrumentation:elasticsearch:elasticsearch-rest-common-5.0:library"))
```

Comment:
is the idea with common modules that we would always add a version suffix for the lowest version number that it is used by?